src/index.ts

```diff
@@ -149,9 +148,10 @@ export class DeviceMock extends EventTarget implements BluetoothDevice {
 
     private serviceMocks: { [service: string]: PrimaryServiceMock } = {};
 
-    constructor(public name: string, private services: BluetoothServiceUUID[]) {
+    constructor(public name: string, private services: BluetoothServiceUUID[], public id: string) {
         super();
         this.gatt = new GattMock(this);
+        this.id = id;
```

Comment:
This line is redundant, as declaring the constructor parameter as `public` automatically assigns its value to the instance field